src/server.py

```diff
@@ -16,13 +18,65 @@ def server():
         conn, addr = server.accept()
         buffer_length = 32
         message_complete = False
-        part = b''
+        full_mes = b''
         while not message_complete:
             partial_mes = conn.recv(buffer_length)
-            part += partial_mes
+            full_mes += partial_mes
             if len(partial_mes) < buffer_length:
                 message_complete = True
-        print(u'received message: ', part.decode('utf8'))
-        conn.sendall(part)
+                full_mes_decoded_to_unicode = full_mes.decode('utf8')
+        print(u'request:\r\n', full_mes_decoded_to_unicode)
+        conn.sendall(response_ok())
         conn.close()
         server.close()
+
+
+def response_ok():
+    """Generate response_ok."""
+    lines = []
+    first_line = 'HTTP/1.1 200 OK'
+    lines.append(first_line)
+    headers = {}
+    body = 'Success!'
+    headers['Content-Type'] = 'text/plain; charset=utf-8'
+    headers['Content-Length'] = str(len(body.encode('utf8')))
+    now = email.utils.formatdate(usegmt=True)
+    headers['Date'] = now
+    header_lines = [': '.join(item) for item in headers.items()]
+    lines.extend(header_lines)
+    lines.append('')
+    lines.append(body)
+    CRLF = '\r\n'
+    response = CRLF.join(lines)
+    response = response.encode('utf8')
+    return response
+
+
+def response_deconstructor(response):
+    """Deconstruct response into basic components.
+    Return protocol, status, status message,
+    headers as dictionary, length of body, number of components
+    separated by first empty line."""
+    response_msg = response
+    uresponse = response_msg.decode('utf8')
+    CRLF = '\r\n'
+    uresponse.split(CRLF)
+    first_pass = uresponse.split(CRLF+CRLF, 1)
```

Comment:
You've already split `uresponse` once on the previous line.  But you didn't use the result of that.  It's good you didn't, since this line is the right way to do the first pass, but you should delete that useless code on line 63 :)